rc/core/sh.kak

```diff
@@ -33,10 +33,10 @@ addhl -group /sh/comment fill comment
 }
 
 addhl -group /sh/code regex [\[\]\(\)&|]{1,2} 0:operator
-addhl -group /sh/code regex (\w+)= 1:identifier
+addhl -group /sh/code regex (\w+)= 1:attribute
```

Comment:
Maybe I got carried away changing this, but it makes highlighting so much nicer:
![](https://my.mixtape.moe/jaaegd.png)

Maybe this shouldn't changed, in hindsight.